src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs

```diff
@@ -107,6 +108,7 @@ internal static class EventLogCollector
                 "vstest.executionengine.clr20.exe",
                 "VSTest.executionEngine.exe",
                 "VSTest.executionEngine.x86.exe",
```

Comment:
While you're here can you delete vbc2.exe. The csc2.exe has already been deleted.

Reply:
➡️ Submitted in #33520